docs/_documentations/importing-existing-projects.md

```diff
@@ -31,13 +31,13 @@ MicroProfile projects are Java applications that are deployed to WebSphere Liber
 
 ### Instructions
 
-Avoid copying files from the project’s Maven target folder as part of any Dockerfile instructions because Codewind builds your project within the container. The application builds against the same runtime that is used in production to avoid inconsistencies between development and production environments.
+Avoid copying files from the project’s Maven target folder to any Dockerfile instructions because Codewind builds your project within the container. The application builds against the same runtime that is used in production to avoid inconsistencies between development and production environments.
 
 ### Pre-import instructions
 
 MicroProfile projects must be configured to build by using Maven.
 
-Tip: For an example of a working MicroProfile application, create a new MicroProfile project in Codewind. You can see the `pom.xml` file contents and use the file as a template when you configure your project's `pom.xml` file.
+**Note:** for an example of a working MicroProfile application, create a new MicroProfile project in Codewind. You can see the `pom.xml` file contents and use the file as a template when you configure your project's `pom.xml` file.
```

Comment:
After terms like "Note:", begin the sentence with a capital letter. In this instance, "For" needs capitalization.